src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -169,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const isCurrentPermTypeDelegated = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+
     if (!panel) {
       columns.push(
         {
           key: 'consentDescription',
           name: messages.Description,
           fieldName: 'consentDescription',
           isResizable: true,
-          minWidth: (tokenPresent) ? 400 : 600,
-          maxWidth: (tokenPresent) ? 600 : 1000,
+          minWidth: (tokenPresent) ? ((isCurrentPermTypeDelegated) ? 400 : 800) : 600,
+          maxWidth: (tokenPresent) ? ((isCurrentPermTypeDelegated) ? 600 : 1300) : 1000,
```

Comment:
```suggestion
          minWidth: (tokenPresent) 
              ? (isCurrentPermTypeDelegated ? 400 : 800) 
              : 600,
          maxWidth: (tokenPresent) 
              ? (isCurrentPermTypeDelegated ? 600 : 1300) 
              : 1000,
```
I think that 2 ternary operators in a row becomes a bit more difficult to notice unless you distinguish them; With this change, I just made the first ternary more apparent, and kept the second ternary the same.

Also, I feel like having the parenthesis around `tokenPresent` makes sense visually because it is an indicator that it is a boolean for the ternary, but since `isCurrentPermTypeDelegated` is already a fully parenthesis-encapsulated ternary, having parenthesis around it feels more difficult to read.

Reply:
agreed!

Reply:
Thanks for the suggestion @AyuAi, we weren't really sure what to do with this!

Reply:
resolved in https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/16/commits/55666db4d0964581599d5f907b355fb066034d67